samples/react-native/src/utils.ts

```diff
@@ -1,7 +1,37 @@
+import { LaunchArguments } from 'react-native-launch-arguments';
+import BuildConfig from 'react-native-build-config';
+import { Platform } from 'react-native';
+
 export function logWithoutTracing(...args: unknown[]) {
   if ('__sentry_original__' in console.log) {
     console.log.__sentry_original__(...args);
   } else {
     console.log(...args);
   }
 }
+
+export function shouldUseAutoStart(): boolean {
+  if (Platform.OS === 'android') {
+    return !!(
+      BuildConfig as {
+        SENTRY_DISABLE_NATIVE_START?: boolean;
+      }
+    ).SENTRY_DISABLE_NATIVE_START;
+  } else if (Platform.OS === 'ios') {
+    const args = LaunchArguments.value<{
+      sentrydisablenativestart?: boolean;
+    }>();
+    return !!args.sentrydisablenativestart;
+  } else {
+    return false;
+  }
```

Comment:
I: Since each if condition returns a value, we could use if on each case and leave return false outside an if condition, what do you think?